### DISCUSSION TITLE
Feature Request: Control dark/bright based on room illuminance

### DISCUSSION THREAD

**Discussion by zazu-sf** [jseidl/magic-areas#293] [Ideas / Feature Requests]
First of all, thank you very much for Magic Areas and the great work. I love this custom component!

At the moment you can control the status of dark/bright with the help of entities. It would be great if Magic Areas could do this automatically. If you have illumination sensors in the room, you know the brightness. Now you could set a lux value in the configuration in the room. As soon as the value falls below this, the status switches from bright to dark and vice versa.

**Comment by jseidl** [comment]
This is a duplicate from https://github.com/jseidl/hass-magic_areas/discussions/182, please upvote that one so I can track your interest on it!
